src/elements/LoadingBar/LoadingBar.less

```diff
@@ -0,0 +1,15 @@
+.ui-loading-bar-wrapper {
+	position: relative;
+	height: 16px;
```

Comment:
Again following the pattern, I think height and margin-bottom should be abstracted into variables, like
```
@ui-loading-bar-wrapper-height: @ui-small; // @ui-small is 16px
@ui-loading-bar-margin-bottom: 20px;

...

height: @ui-loading-bar-wrapper-heigh;

...

margin-bottom: @ui-loading-bar-margin-bottom
```